backend/src/users/controllers.ts

```diff
@@ -563,6 +562,58 @@ const editHours = async (req: Request, res: Response) => {
   }
 };
 
+/**
+ * Returns sorted Users based on one key
+ * @returns promise with user or error
+ */
+const getUsersSorted = async (req: Request, res: Response) => {
+  // #swagger.tags = ['Users']
+  const query = req.query.sort as string;
+  const querySplit = query.split(":");
+  const key: string = querySplit[0];
+  const order = querySplit[1];
+
+  try {
+    if (key == "email") {
+      const users = await prisma.user.findMany({
+        orderBy: [{ email: order == "asc" ? "asc" : "desc" }],
+      });
+      res.status(200).json(users);
+    } else if (key == "hours") {
+      const users = await prisma.user.findMany({
+        orderBy: [{ hours: order == "asc" ? "asc" : "desc" }],
+      });
+      res.status(200).json(users);
+    } else if (key == "firstName") {
+      const users = await prisma.user.findMany({
+        orderBy: {
+          profile: {
+            firstName: order == "asc" ? "asc" : "desc",
+          },
+        },
+        include: {
+          profile: true,
+        },
+      });
+      res.status(200).json(users);
+    } else if (key == "lastName") {
+      const users = await prisma.user.findMany({
+        orderBy: {
+          profile: {
+            lastName: order == "asc" ? "asc" : "desc",
+          },
+        },
+        include: {
+          profile: true,
+        },
+      });
+      res.status(200).json(users);
+    }
+  } catch (error) {
+    res.status(500).json({ error: (error as Error).message });
```

Comment:
We should ideally have `successJson` and `errorJson` functions for these responses